doc/ale-graphql.txt

```diff
@@ -13,5 +13,10 @@ You will need the GraphQL ESLint plugin installed for this to work.
 ===============================================================================
 gqlint                                                     *ale-graphql-gqlint*
 
+===============================================================================
+prettier                                                   *ale-graphql-prettier*
+
+See |ale-graphql-prettier| for information about the available options.
```

Comment:
You might want to write `ale-javascript-prettier` here so this sends you to the documentation for the `prettier` fixer.